pkg/api/v1/conditions/client/client_test.go

```diff
@@ -325,7 +332,16 @@ func TestFirmwareInstall(t *testing.T) {
 				AssetID: serverID,
 			},
 			mockStore: func(r *store.MockRepository) {
-				r.On("CreateMultiple", mock.Anything, serverID, mock.Anything, mock.Anything).
+				r.On(
+					"CreateMultiple",
+					mock.Anything,
+					serverID,
+					mock.Anything, // I haven't figured a way pass in a variable list of conditions to this r.On mock method
```

Comment:
Since `CreateMultiple` accepts upto n Conditions, I was trying to get the mock `r.On` method to accept a variable list of `*rctypes.Condition` such that the test doesn't have to be updated if the Condition composition changes.

Reply:
I did some searching yesterday. Mockery does support variadic methods I think.

Reply:
Thanks, added the fix in https://github.com/metal-toolbox/conditionorc/pull/221/commits/1484fff561d2aea85deb7a8bf439ee221422dab3